### PR TITLE
Middleware/Links: add OPEN_SITE

### PR DESCRIPTION
**Pull request by samouri**
We already send out OPEN_COMMENT and OPEN_POST.  It only naturally follows that we should also have OPEN_SITE.  

Question: are there any others that are missing that should open in Reader?

### CODE REVIEW

**Comment by samouri** [comment]
To Test in Calypso:
0. go to master branch and make sure you have latest
1. npm install 'Automattic/notifications-panel#add/open-site-action'
2. make start

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifications-panel",
-  "version": "1.2.1",
+  "version": "1.3.0",
```
Comment (samouri) [inline package.json]: it changes the api. 

hypothetically if a client were expecting OPEN_LINKs they will be unpleasantly surprised that now they are getting OPEN_SITE

Comment (dmsnell) [inline package.json]: yeah, make your call. no client currently would be surprised I think

Comment (dmsnell) [inline package.json]: to be consistent though, we just made this decision in a recent PR along these "wrong" rules

**Pull request merged**